### PR TITLE
Hydroponics Spray Fix

### LINKED ISSUE #14049: Pest spray is misclassified and misdisplayed
**Issue Description**:
Two very minor potential bugs with the "pest spray" item (`/obj/item/reagent_containers/spray/pestspray`):
1. The item's icon is displayed with an offset to the upper-left. This looks fine when it is in the environment, but looks strange in inventory slots.
2. Unlike fertilizer bottles and botany tools, pest spray cannot be stored in the botany belt (`/obj/item/storage/belt/botany`).

**Why is this bad/What are the consequences**:
These are very minor nitpicks, but the fact that this object seems wonky in multiple ways suggests there may be some larger underlying bug.

**Steps to reproduce the problem**:
1. Find or spawn the pest spray item.
2. Equip it to observe the icon is off-center.
3. Try to store it in a botany belt to observe it is not classified as a botany tool.

**When did the problem start happening**:
I noticed it when I began playing Botanist a couple weeks ago.

### PR DESCRIPTION
**Pull request by ExusA**
## What Does This PR Do
Fixes #14049 by:
1. Re-centers all hydro sprays, making them look less weird
2. Lets you put both weed killer and pest killer on a hydro belt

## Why It's Good For The Game
The de-centered sprays looked super weird, and you should be able to put hydro equipment like pestkiller on a hydro belt. You can already put both items on the hydro apron anyway, so it makes sense

## Images of changes

before:
pestspray_uncentred

after:
pestspray_centred

## Changelog
:cl:
fix: centers hydroponic sprays' sprites
tweak: makes hydroponics sprays able to be placed in the hydroponics belt
/:cl:

### CODE REVIEW

**Comment by ExusA** [comment]
oh jeez, it's beyond me how that happened. fixing now

**Comment by ExusA** [comment]
sorry about that, i'm not much good with github :pensive:

**Comment by ExusA** [comment]
aaa wrong branch AGAIN

File: code/game/objects/items/weapons/storage/belt.dm [hunk 1]
```diff
@@ -204,8 +204,13 @@
...
 		/obj/item/seeds,
 		/obj/item/wirecutters,
+		/obj/item/wrench,
+   		/obj/item/reagent_containers/spray/weedspray,
+    	/obj/item/reagent_containers/spray/pestspray,
         /obj/item/wrench,
-	)
+        /obj/item/reagent_containers/spray/weedspray,
+        /obj/item/reagent_containers/spray/pestspray
+        )
```
Comment (AffectedArc07) [inline code/game/objects/items/weapons/storage/belt.dm:213]: The indents here seems really funky. Did you mix tabs and spaces?

Comment (ExusA) [reply code/game/objects/items/weapons/storage/belt.dm:213]: oh lordy. did that trying to fix a merge conflict, aagh

Comment (ExusA) [reply code/game/objects/items/weapons/storage/belt.dm:213]: fixing now

File: code/game/objects/items/weapons/storage/belt.dm [hunk 2]
```diff
@@ -204,8 +204,10 @@
 		/obj/item/flashlight/pen,
 		/obj/item/seeds,
 		/obj/item/wirecutters,
-        /obj/item/wrench,
-	)
+		/obj/item/wrench,
+		/obj/item/reagent_containers/spray/weedspray,
+		/obj/item/reagent_containers/spray/pestspray
+        )
```
Comment (AffectedArc07) [inline code/game/objects/items/weapons/storage/belt.dm:210]:
```suggestion
		)
```
We indent with tabs here

**Review by AffectedArc07 — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weapons/storage/belt.dm (hunk 2 above)
Comment (ExusA) [reply code/game/objects/items/weapons/storage/belt.dm:210]: i.. have no words. i SWEAR i indented with tabs originally. this pr has a curse on it 😒

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**